docs/docs/Components/components-vector-stores.md

```diff
@@ -66,6 +66,40 @@ For more information, see the [DataStax documentation](https://docs.datastax.com
 | vector_store | Vector Store | Built Astra DB vector store |
 | search_results | Search Results | Results of the similarity search as a list of Data objects |
 
+
+## AstraDB Graph vector store
+
+This component implements a Vector Store using AstraDB with graph capabilities.
```

Comment:
For more information, see the [DataStax documentation](https://docs.datastax.com/en/astra-db-serverless/tutorials/graph-rag.html).